src/App.js

```diff
@@ -1,10 +1,12 @@
 import { BrowserRouter as Router } from 'react-router-dom';
 import AnimatedRoutes from './AnimatedRoutes';
 import './App.css';
+import Navbar from './components/Layout/Navbar';
 
 function App() {
   return (
    <Router>
+    <Navbar />
```

Comment:
Place the Navbar into appropriate Layout component that will be the base of all the viewable pages on the application